miden/tests/integration/stdlib/math/ecgfp5/scalar_field.rs

```diff
@@ -132,7 +132,7 @@ impl Scalar {
     /// Given a scalar in radix-2^32 form, this routine converts it to Montgomery form
     ///
     /// Inspired by https://github.com/itzmeanjan/secp256k1/blob/37b339d/field/scalar_field_utils.py#L235-L242
-    fn to_mont(&self) -> Self {
+    fn as_mont(&self) -> Self {
```

Comment:
I think the `to_mont` is more descriptive. We usually use `as_` when it's a borrow; meaning we just pass around some reference to something. There could be an exception if the type is like a `u64` where you don't pass a reference but instead pass the copied value (it's cheaper than a pointer), but the value should already exist somewhere for this exception to be met.

In this case, we perform a full conversion of the type into a new instance. Since this is fixed cost, it should be `to_`, whereas if it was variable cost, it should be `into_`.

Reply:
Thanks! Yeah, that's in line with my thoughts, so I think we should keep it as it was.